lib/include/jxl/gain_map.h

```diff
@@ -0,0 +1,110 @@
+/* Copyright (c) the JPEG XL Project Authors. All rights reserved.
+ *
+ * Use of this source code is governed by a BSD-style
+ * license that can be found in the LICENSE file.
+ */
+
+/** @addtogroup libjxl_metadata
+ * @{
+ * @file gain_map.h
+ * @brief Utility functions to manipulate jhgm (gain map) boxes.
+ */
+
+#ifndef JXL_GAIN_MAP_H_
+#define JXL_GAIN_MAP_H_
+
+#include <jxl/color_encoding.h>
+#include <jxl/jxl_export.h>
+#include <jxl/types.h>
+
+#ifndef __cplusplus
+extern "C" {
+#endif
+
+/** Gain map bundle */
+typedef struct {
+  /** version number */
+  uint8_t jhgm_version;
+  /** size of the gain map metadata */
+  uint16_t gain_map_metadata_size;
+  /** pointer to binary blob of gain map metadata (ISO 21496-1) */
+  const uint8_t* gain_map_metadata;
```

Comment:
Something somewhere should clarify that these fields are not owned.

Reply:
added long docstring like we before struct like we have in cms_interface.h